service/rate.go

```diff
@@ -69,11 +69,29 @@ func (h *Service) HandleLinkMessage(req LinksRequest) (string, error) {
 		return "", errWontJoin
 	}
 
-	groupRate, err := h.getRateForGroup(req.Channel, groupID.ID, req.MessageID)
-	if err != nil {
-		if errors.Is(err, errNotInTime) {
-			return "", nil
+	shouldHandleOrder := h.shouldHandleOrder()
+	if !shouldHandleOrder {
+		_, err := h.informEvent(req.Channel, "It's too late for me... I won't track prices for this order :sleeping:", "", req.MessageID)
+		if err != nil {
+			return "", errWontJoin
 		}
+
+		return "", errNotInTime
+	}
```

Comment:
Pay attention in the original function, I don't exit immediately in case shouldHandleOrder is false. The reason is that I still want to monitor the order status and the venue availability even if prices should not be tracked.